docs/modules/data-structures/pages/map-config.adoc

```diff
@@ -41,35 +52,32 @@ YAML::
 hazelcast:
     map:
         default:
-            # backup count
-            backup-count: 1
-            async-backup-count: 0
-            # in-memory format
-            in-memory-format: binary
-            # expiration settings
-            time-to-live-seconds: 60
-            max-idle-seconds: 60
-            # eviction setting
-            eviction:
-                eviction-policy: NONE
-                max-size-policy: PER_NODE
-                size: 0
-            #other map settings as needed
-
+            backup-count: 1 #<1>
+            in-memory-format: BINARY #<2>
+            max-idle-seconds: 3600
+            time-to-live-seconds: 21600
 ----
+<1> We did not want to modify the default backup count. If this line were omitted, we would have no backups.
+<2> You must specify an in-memory format.
 ====
 
-Each configuration option is covered in detail in the relevant section of the documentation. For example, the expiration and eviction settings are discussed in the xref:managing-map-memory.adoc[managing map mempry] section. 
+== Multiple Map Configurations
+
+Your application may require multiple maps with different settings. To accomplish this, you can create multiple named map configurations. The Hazelcast cluster will apply the stored configuration to each map when the map is created, based on the name of the map. If there's no name match, the map will use the settings in the default configuration. 
+
+In this example, we're creating a configuration for the Customers map and setting the in-memory format to OBJECT. When we run our code, Hazelcast will match the map name to the configuration name and apply all the settings included under Customers. 
```

Comment:
Wrap 'Customers' and 'OBJECT' in backticks to make it clear that these refer to parts of code/config.